src/utils/pyproject.ts

```diff
@@ -40,9 +48,21 @@ function parsePyproject(pyprojectContent: string): string | undefined {
   )
     .flat()
     .filter((item: string | object) => typeof item === "string");
-  return getRuffVersionFromAllDependencies(
+  const version = getRuffVersionFromAllDependencies(
     dependencies.concat(optionalDependencies, devDependencies),
   );
```

Comment:
```suggestion
  const allDependencies = dependencies.concat(optionalDependencies, devDependencies, poetryDependencies);
  const version = getRuffVersionFromAllDependencies(
    allDependencies
  );
```
For `poetryDependencies` see below